stanford_courses.features.field_instance.inc

```diff
@@ -1055,7 +1055,7 @@ function stanford_courses_field_default_field_instances() {
         'size' => 60,
       ),
       'type' => 'text_textfield',
-      'weight' => 17,
+      'weight' => 9,
```

Comment:
We're adjusting weights of existing fields here because we're adding a field, yeah?